IdentityCore/xcconfig/identitycore__common__ios.xcconfig

```diff
@@ -5,4 +5,3 @@ LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path
 TARGETED_DEVICE_FAMILY = 1,2
 ENABLE_BITCODE = YES
 SKIP_INSTALL = YES
-ARCHS[sdk=iphoneos*] = $(ARCHS_STANDARD) arm64e
```

Comment:
Just curious, it is an XCode bug that it fails to compile for arm64e? :)

Reply:
Yeah, looks like that :) arm64e was experimental anyway and removed automatically when uploading to App Store or building with Xcode 10.2.